earmuff-jam/mashed
# Address the ability to control user group for a selected event

## Description
- users who create an event will not be allowed to leave that event as it provides good audit trail and also give ownership to the creator
- removed tech debt and refactor code for usability

## Testing steps
- create a new event
- go to sharable modal and assign other users to the event
- attempt to remove yourself from the event ( you should not be able to )
- assume the role of a third person ( log in as a new user ) and try to remove the creator from the sharable modal.

## Expected Outcome
- The creator should not be able to be removed from the selected event



## Screenshots / Visuals

![image](https://github.com/earmuff-jam/mashed/assets/44540219/2ed5c454-1f02-46e3-ad73-90cc89ad0d15)
![image](https://github.com/earmuff-jam/mashed/assets/44540219/a12d9602-bef0-45b3-b08e-052fd38f4d9e)
![image](https://github.com/earmuff-jam/mashed/assets/44540219/c4f11678-1b31-4a04-84ec-46da66c44896)
